scripts/build.sh

```diff
@@ -30,14 +30,17 @@ Flags:
     -r|--runtime-image=<string>          Overrides the image to use for running the workflow.
     -n|--namespace=<string>              The target namespace where the manifests will be applied. Default: current namespace.
     -m|--manifests-directory=<string>    The operator manifests will be generated inside the specified directory. Default: 'manifests' directory in the current directory.
-    -w|--workflow-directory=<string>     Path to the directory containing the workflow's files (the 'src' directory). Default: current directory.
+    -w|--workflow-directory=<string>     Path to the directory containing the workflow's files. For Quarkus projects, this should be the directory containing 'src'. For simple layout, this should be the directory containing the workflow files directly. Default: current directory.
     -P|--no-persistence                  Skips adding persistence configuration to the sonataflow CR.
+    -S|--simple-layout                   Use simple layout (non-Quarkus) where workflow resources are in the project root instead of src/main/resources.
```

Comment:
why simple-layout and not quakus-layout?

Reply:
Sorry, I read `if [[ -n "${args["simple-layout"]:-}" ]]; then` the other way, so why not `serless-layout` instead?